### ISSUE TITLE
asciidoctor links to adoc files

### ISSUE THREAD

**Issue by lesserwhirls** [Unidata/thredds#345]
I thought asciidoctor could handle links to adoc files automagically, but it doesn't (my bad). Currently we try to link to adoc files using something like the following:

```
link:rel/path/to/doc.adoc[text to link]
```

However, we should use the following:

```
<<rel/path/to/doc.adoc,text to link>>
```

otherwise, we simply link to the adoc file itself.

Looks like a job for a regex...which means I should open another ticket, since we now have two problems :-)

**Comment by cwardgar** [comment]
I left some comments about this on the pull request instead of here. Oops. https://github.com/Unidata/thredds/pull/346#issuecomment-168367403

Reopening this for now.

**Comment by lesserwhirls** [comment]
Ok, I've made a pr remove the .adoc file extension and contain a # symbol. Let's see how this does. See PR Unidata/thredds#366

**Comment by cwardgar** [comment]
Fixed by #366.

**Comment by cwardgar** [comment]
Looks like we missed some:
````
[cwardgar@MacBook-Pro ~/dev/projects/thredds2] (5.0.0 %>)
$ egrep -r --include '*.adoc' 'link:.*\.adoc' docs/ | wc -l
     117
````
Reopening.
